### ISSUE TITLE
[Bug] Zanac: title corrupted [sf#93]

### ISSUE THREAD

**Issue by openMSX-import** [openMSX/openMSX#394]
**Reported by manuelbi on 2003-05-17 15:10 UTC**
Zanac shows a corrupted title screen (in the logo) and
does not run on 16kB MSX machines (like National
CF-2000), while it should. It does run with 32kB.

**Comment by openMSX-import** [comment]
**Commented by manuelbi on 2004-10-29 12:36 UTC**
Logged In: YES 
user_id=78178

I don't know what causes it, but it works fine now on 16kB
MSX machines, like the CF-2000 and the CF-1200.

**Comment by openMSX-import** [comment]
**Updated by manuelbi on 2004-10-29 12:36 UTC**
- **summary**: Zanac: title corrupted and doesn't work on 16kB MSX --> Zanac: title corrupted

**Comment by openMSX-import** [comment]
**Commented by m9710797 on 2009-06-04 21:44 UTC**
Finally, after 5 years we finally fixed this bug!!

It's related to switching the IE0 bit in the VDP register while there is a pending VBANK IRQ. See commit message of  revision 9983 for more details.

It's actually also a bug in the zanac program (but not triggered on a real MSX). It's writing to VRAM with z80 interrupts enabled (EI). But the interrupt routine changes the VRAM write pointer. Before this bug was fixed, there really happened an IRQ at the wrong moment in time in openMSX.
